submissions/reactjs/assignment3.MD

```diff
@@ -42,5 +42,5 @@
 | Bharvi Valand                  | 2303031087002        |                   |             |
 | Neeti Darji                    | 2303031087080        |                   |             |
 | Suhani Tandel                  | 2203031080160        |                   |             |
-| Rishabh Rai                    | 2203031080114        |                   |             |
+| Rishabh Rai                    | 2203031080114        |    https://github.com/Rishabhrai29/classReact/tree/main/classReact/src       |   https://github.com/Rishabhrai29/classReact/tree/main/classReact          |
```

Comment:
Please follow the repository structure mentioned in the problem statement document